library/tiqr/Tiqr/StateStorage/Pdo.php

```diff
@@ -101,7 +101,7 @@ private function cleanExpired(): void {
             $sth->execute(array(time()));
             $deletedRows=$sth->rowCount();
             $this->logger->notice(
-                sprintf("Deleted %i expired keys", $deletedRows)
+                sprintf("Deleted %d expired keys", $deletedRows)
```

Comment:
:musical_note: 
A is for apple
B is for balloon
C is for crayons 
`%d` is for `digit` :) 

Just so you never have to forget